Makefile

```diff
@@ -34,8 +34,7 @@ go-get:
 go-test:
 	@echo "  >  Running ginkgo test suites..."
 	# can replace with a recursive command ginkgo suites are defined for all packages
-	ginkgo grpc inventory
-	ginkgo grpc network
+	ginkgo grpc inventory storage network
```

Comment:
see if any changes needed in `mock-generate:`

Reply:
I have not created any interfaces yet